_posts/2019-03-13-activeadmin-vs-administrate.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "ActiveAdmin vs Administrate"
```

Comment:
@rdormer hm, maybe you can use a different title that doesn't mention the names of the gems?